server/src/main/java/com/thoughtworks/go/server/web/BackupFilter.java

```diff
@@ -68,6 +68,12 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
         }
 
         String url = ((HttpServletRequest) request).getRequestURI();
+
+        if (isFilesAPI(url)) {
```

Comment:
@ketan Rethinking this, I'm not sure if this is a good idea. Can't calls [such as this](https://github.com/gocd/gocd/blob/aa4c0d44fd65b2a7c5c694f923f202cb6f4ecc7c/server/src/main/java/com/thoughtworks/go/server/newsecurity/providers/AbstractPluginAuthenticationProvider.java#L121) affect the DB, while a backup is in progress?

Reply:
Fair point, I'll yank that commit out.

Reply:
Yanked. Can someone take a look?